packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -151,6 +154,24 @@ class AppContext(
   val filePermission: FilePermissionModuleInterface?
     get() = legacyModule()
 
+  /**
+   * Provides access to the scoped directories from the legacy module registry.
+   */
+  private val appDirectories: AppDirectoriesModuleInterface?
```

Comment:
Not sure if this should be private (as this currently can be used for nothing else then the properties surfaced below)

Reply:
Let's leave it private for now ;)